secrets/json_secrets.go

```diff
@@ -52,6 +52,13 @@ func WriteComposite(composite config.CompositeSecretFile, cache briefcase.Secret
 		}
 	}
 
+	if composite.Secrets.Owner != nil {
+        err = os.Chown(composite.Filename, composite.Secrets.Owner , composite.Secrets.Owner)
```

Comment:
I think os.Chown takes a integer GID + UID, you'll probably need to do a lookup for those.